test/integration/features/basic.feature

```diff
@@ -102,15 +102,6 @@ Feature: Basic
       hello
       """
 
```

Comment:
Can you also change:

```
  Scenario: Starting Minishift
    Given Minishift has state "Does Not Exist"
     When executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT" succeeds
     Then Minishift should have state "Running"
```

As this is where we set the variables which are tested in this part.